charts/templates/coordinator.yaml

```
@@ -7,7 +7,7 @@ metadata:
     app.kubernetes.io/component: coordinator
     app.kubernetes.io/created-by: {{ .Values.global.createdBy }}
     app.kubernetes.io/name: coordinator
-    app.kubernetes.io/part-of: marblerun
+    app.kubernetes.io/part-of: {{ .Release.Namespace }}
```

```suggestion
    app.kubernetes.io/part-of: marblerun
```
There already is a label to filter by namespace (`{{ .Values.global.coordinatorNamespaceLabel }}: {{ .Release.Namespace }}`)
This label is supposed to allow application level filter, its not scoped to a namespace.

---

You are right, nice catch!